src/atlas_assistant/api.py

```diff
@@ -53,6 +60,17 @@ def create_access_token(self, settings: Settings) -> str:
         )
 
 
+class ChatRequest(BaseModel):
+    """A chat request"""
+
+    query: str
+    """The question from the user."""
```

Comment:
We have a decision to make on whether we let a user configure state (what region, attachments, etc) incrementally within their session via repeated calls to the API, or let the frontend track all the state and then send it all with this chat request.